ovos_utils/gui.py

```diff
@@ -461,7 +501,14 @@ class GUIInterface:
     """
 
     def __init__(self, skill_id, bus=None, remote_server=None, config=None):
-        self.config = config or Configuration().get("gui", {})
+        if not config:
```

Comment:
For a future release, consider making `config` a required arg and dropping ovos-config fallback